docs/interest-filtering.md

```diff
@@ -0,0 +1,105 @@
+# Interest filters
+
+## `pantheon.ei.localized_terms`
+
+Modify terms before they are localized.
+
+### Parameters
+
+$terms __(array)__ An array of terms to modify.
+
+### Example
+
+Add term to array:
+```bash
+add_filter( 'pantheon.ei.localized_terms', 'pantheon_ei_add_term' );
+function pantheon_ei_add_term( $terms ) {
```

Comment:
I like adding strict typing to documentation. I think it helps make the examples more clear about the expected inputs and outputs. I know this is also documented in the parameters, but it reinforces that and also establishes best practices. 🔢

```suggestion
function pantheon_ei_add_term( array $terms ) : array {
```